src/web/backend/ldap/models.py

```diff
@@ -39,13 +40,37 @@ def __repr__(self):
     def create(self, password):
```

Comment:
Changed it to call only self methods. However, left logic for processing Exceptions of methods for creating rocketchat, nextcloud entities in `create` method, to leave it possible to debug and work with those methods separately. Is that ok ?

Reply:
I would prefer those separate methods to do the exception handling. I am not opposed to one-line methods, but it is best if the method performs a set of tasks, like exception handling and that it can pass results.
If it helps, it is OK to introduce a mid-level methods that would perform those tasks - like that, `create` will have a truly descriptive body, and the reusable low-level methods will stay there, available for re-use.

Reply:
Agree. Removed exception handling from create methods